packages/desktop-client/src/components/admin/UserAccess/UserAccess.tsx

```
@@ -0,0 +1,345 @@
+      .map(user => ({
+        ...user,
+        displayName: user.displayName || user.userName,
+      }))
+      .sort(sortUsers);
+
+    setAllAccess(loadedAccess);
+    return loadedAccess;
+  }, [cloudFileId, setLoading]);
+
+  const loadOwner = useCallback(async () => {
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)